gnosis/safe/tests/test_safe.py

```diff
@@ -707,6 +707,26 @@ def test_retrieve_is_message_signed(self):
         safe_tx.execute(tx_sender_private_key=self.ethereum_test_account.key)
         self.assertTrue(safe.retrieve_is_message_signed(message_hash))
```

Comment:
Please update this old test to make sure `self.assertFalse(safe.retrieve_is_message_signed(message_hash))` before signing

Reply:
Done https://github.com/safe-global/safe-eth-py/pull/736/commits/bd147e60cf3ae3316271f9ed3150d9601939e8f8